modules/netdevedit.php

```diff
@@ -758,6 +754,28 @@
         $netdev['model'] = $netdev['modelid'];
     }
 
+    $attachmenttype = 'netdevid';
+    $attachmentresourceid = $id;
+    $SMARTY->assign('attachmenttype', $attachmenttype);
+    $SMARTY->assign('attachmentresourceid', $attachmentresourceid);
+
+    $filecontainers = array(
+        'netdevid' => array(
+            'id' => $id,
+            'prefix' => trans('Attachments of the device:'),
+            'containers' => $LMS->GetFileContainers('netdevid', $id),
+        ),
+        'netdevmodelid' => array(
+            'id' => $netdev['modelid'],
+            'prefix' => trans('Attachments of the model:'),
+            'containers' => $LMS->GetFileContainers('netdevmodelid', $netdev['model']),
+        ),
+    );
+    $SMARTY->assign('filecontainers_for_netdevice', $filecontainers['netdevid']);
+    $SMARTY->assign('filecontainers_for_netdevmodel', $filecontainers['netdevmodelid']);
```

Comment:
To zwinąć do:
```php
$SMARTY->assign('filecontainers', $filecontainers);
```

i prawdopodobnie przenieść gdzieś pod koniec attachments.php.

Wszędzie w szablonach html, gdzie odwołujemy się do np. `$filecontainers_for_netdev` zamienić na `$filecontainers.netdevid`.